.github/workflows/differential-shellcheck.yml

```diff
@@ -50,3 +50,11 @@ jobs:
           name: Differential ShellCheck SARIF
           path: ${{ steps.ShellCheck.outputs.sarif }}
           retention-days: 7
+
+      - if: ${{ always() }}
```

Comment:
nit: `${{ ... }}` for `if` is not necessary, e.g. like here: https://github.com/aufover/aufover-benchmark/blob/d2278c8569c59e44c088ff66e5e51d89ac60805e/.github/workflows/fedora.yml#L91

Reply:
You are right; I'll drop it in the follow-up. Thank you

Reply:
Addressed in:
- https://github.com/redhat-plumbers-in-action/differential-shellcheck/pull/401